FORMAT.md

```diff
@@ -212,10 +226,35 @@ Note: the HRDF file is ill-formed and should generate a parsing error if both `m
 <actuator type="X5-9" mass_offset="0.2"/>
 ```
 
+## Output element
+
+The `output` element is a special child element used to define the connection points for brackets and rigid bodies with multiple output interfaces.  It is unnecessary for pure kinematic chains. It may only be present as the child of a `bracket` with multiple output interfaces or a `rigid-body` element.  For `bracket` and `rigid-body` elements with a single output interface, omitting the `output` element and using a chain of robot model elements is preferred (see below, "Connecting Robot Model Elements").
+
+The `output` element can only contain [`robot element`](robot-elements) subelements.
+
+**Optional attributes:**
+
+- `rot` (rotation matrix): the orientation of the output interface frame relative to the input interface. Defaults to identity or the value defined by the parent `bracket` or `rigid-body` element. This attribute will cause a parsing error if parent element of the `output` element is not a `rigid-body`.
+- `trans` (translation vector, m): The position the output interface frame.  Defaults to (0,0,0). Defaults to identity or the value defined by the parent `bracket` or `rigid-body` element. This attribute will cause a parsing error if parent element of the `output` element is not a `rigid-body`.
+
+**Notes/Constraints:**
+
+In a bracket, the output interfaces are inherently defined, and so the rotations and translations are not permitted to be changed.
+
```

Comment:
I'm not proud of the "you can override this if the output is a child of a rigid body but not if it is the child of a bracket".  It is a little confusing and not really definable as a schema constraint.

However, I think this is OK; we don't have full validation possible with the XML format here (see the floating point formula or the restrictions on `mass` and `mass_offset` attributes at the same time, for example).  And I do like keeping the bracket outputs fully defined by the bracket itself -- this matches our behavior elsewhere, where we can override mass/intertia/COM properties, but not the actual kinematic information.

Reply:
How about the output attribute serves as the first output, and you can define additional outputs as child nodes? If no attribute is specified, the child nodes start at zero.

Even a link can have multiple outputs, e.g., when you mount a gripper actuator. Or would you prepend a multi-output-body that the contains the link and the gripper?

Reply:
My comment was specifically about overriding the `rot` and `trans` elements in certain cases, but I'm happy to discuss other aspects further.

For the link + gripper actuator -- I don't see that other actuator as part of the kinematics, b/c it's output frame isn't really relevant.  Honestly, links in general are a bit of syntactic sugar; they overlap with brackets conceptually (a named/defined rigid body), but just provide some reduction of enum-explosion when it comes to output directions/etc.